src/components/Repository.js

```diff
@@ -98,8 +98,11 @@ function Repository({match}) {
       .finally(() => setIsForkLoading(false));
   };
 
+
   const user = getUserFromJwt(Config.auth);
```

Comment:
This needs to be mocked in the test @filiptronicek 

similar to https://github.com/open-sauced/open-sauced/blob/master/src/tests/App.test.js#L24

Reply:
Turns out this line should not be here. I did some work on this PR and got it working with mock and a spyOn. There is a rule I live by, which is if a test is hard to write the code can be refactored. 

I am going to open a PR to remove this.